x-pack/plugins/alerting/README.md

```diff
@@ -766,6 +766,7 @@ This factory returns an instance of `Alert`. The `Alert` class has the following
 
 |Method|Description|
 |---|---|
+|getUuid()|Get the UUID of the alert.|
```

Comment:
Should line 786 also include `alertUuid` in the list of `Templating Actions`?

Reply:
Rather than update a probably already incomplete list, I added a link to the public doc, which I believe we keep up-to-date a little better.